### DISCUSSION TITLE
dnspod issue: zone not found

### DISCUSSION THREAD

**Discussion by huiyifyj** [go-acme/lego#1787] [Q&A] [answered]
> My domain is punycode domain, and the problem occurs when I apply with docker.

```log
2022/12/17 10:48:23 Could not obtain certificates:
	error: one or more domains had a problem:
[*.xn--<xxx>.com] [*.xn--<xxx>.com] acme: error presenting token: zone xn--<xxx>.com. not found in dnspod for domain _acme-challenge.xn--<xxx>.com.
```
![log-info](https://user-images.githubusercontent.com/32027253/208238440-5166a7ac-de22-477d-9ca0-8338c39213b8.png)

I try to `tencentcloud`, but still fail.
```log
2022/12/17 11:41:40 Could not obtain certificates:
	error: one or more domains had a problem:
[*.xn--<xxx>com] [*.xn--<xxx>.com] acme: error presenting token: tencentcloud: failed to get hosted zone: zone xn--<xxx>.com. not found in dnspod for domain _acme-challenge.xn--<xxx>.com.
```

**Comment by ldez** [comment]
have you tried to not puny domain?

**Reply by huiyifyj** [reply]
I only have this domain under my account, so I can't confirm whether because of punycode. 😔

**Comment by huiyifyj** [comment, marked as answer]
It is indeed the cause of punycode domain.

I try to https://github.com/go-acme/lego/pull/1788, and build binrary, I reapply successfully.

![image](https://user-images.githubusercontent.com/32027253/208306695-4ba9e3fe-38ca-4a6f-9714-2aa78ef30855.png)
